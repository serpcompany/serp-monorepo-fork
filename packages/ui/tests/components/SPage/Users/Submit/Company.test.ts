/* eslint-disable @typescript-eslint/no-unused-vars, no-unused-vars  */

import { mockNuxtImport } from '@nuxt/test-utils/runtime';
import { describe, expect, it, vi } from 'vitest';
import { ref } from 'vue';
import SubmitCompany from '../../../../../components/SPage/Users/Submit/Company.vue';
import ComponentRender from '../../../../componentRender';

// Module level variables for mutable state
let routeMock_: Record<string, unknown> = { query: {} };
let session_: unknown;
const s3Object_ = {
  upload: async () => '/api/s3/query/dummy'
};
const companyCategoriesData_ = [
  { id: 1, name: 'Tech', slug: 'tech' },
  { id: 2, name: 'Finance', slug: 'finance' }
];
const companySubmissionsData_ = {
  id: 123,
  approved: false,
  name: 'Test Company',
  domain: 'testcompany.com',
  pricing: 'Paid',
  tags: 'innovative, startup',
  oneLiner: 'Best company ever',
  description: 'Detailed description',
  categories: [1, 2],
  logo: 'logo.png',
  uuid: 'uuid-123',
  isPriority: false
};

// Mock all Nuxt imports
mockNuxtImport('useRoute', () => () => routeMock_);
mockNuxtImport('useUserSession', () => () => session_);
globalThis.useS3Object = () => s3Object_;
mockNuxtImport(
  'useCompanyCategories',
  () => async () => companyCategoriesData_
);
mockNuxtImport(
  'useCompanySubmissions',
  () => async (id: string) => companySubmissionsData_
);
mockNuxtImport('useToast', () => () => ({
  add: vi.fn()
}));
mockNuxtImport('useRuntimeConfig', () => () => ({
  public: {
    cloudflareR2PublicUrl: 'https://dummy.cloudflare/'
  }
}));

describe('SPage/Users/Submit/Company Snapshot', () => {
  const scenarios: [
    string,
    { session: unknown; route: Record<string, unknown> }
  ][] = [
    [
      'New Submission (no id)',
      {
        session: {
          loggedIn: ref(true),
          user: ref({ email: 'test@test.com' })
        },
        route: { query: {} }
      }
    ],
    [
      'Existing Submission (with id)',
      {
        session: {
          loggedIn: ref(true),
          user: ref({ email: 'test@test.com' })
        },
        route: { query: { id: 'existing' } }
      }
    ]
  ];

  it.each(scenarios)(
    'renders %s correctly',
    async (desc, { session, route }) => {
      // Update the mutable session and route for the current scenario
      session_ = session;
      routeMock_ = route;

      const html = await ComponentRender(
        `SPage/Users/Submit/Company ${desc}`,
        {},
        SubmitCompany
      );
      expect(html).toMatchSnapshot();
    }
  );
});
