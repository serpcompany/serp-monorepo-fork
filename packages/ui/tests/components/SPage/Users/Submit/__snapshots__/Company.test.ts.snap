// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SPage/Users/Submit/Company Snapshot > renders Existing Submission (with id) correctly 1`] = `
"<div data-v-8dd619f3="" class="flex-col lg:grid lg:grid-cols-10 lg:gap-10 flex min-h-screen items-center justify-center">
  <!--v-if-->
  <div class="lg:col-span-10">
    <main data-v-8dd619f3="" class="min-h-[calc(100vh-var(--ui-header-height))] py-20">
      <div data-v-8dd619f3="" class="rounded-lg bg-default ring ring-default divide-y divide-default mx-auto max-w-2xl p-8">
        <!--v-if-->
        <div class="p-4 sm:p-6">
          <uheading data-v-8dd619f3="" level="2" class="mb-6 text-center text-2xl font-semibold"> Submit Company </uheading>
          <div data-v-8dd619f3="" class="mb-6 text-center">
            <p data-v-8dd619f3="" class="text-sm text-gray-500"> Your company is already submitted. You can edit the details below. </p>
            <div data-v-8dd619f3=""><span data-v-8dd619f3="" class="font-medium inline-flex items-center text-xs px-2 py-1 gap-1 rounded-md bg-error text-inverted"><!--v-if--> Not Verified <!--v-if--></span><button type="button" class="rounded-md font-medium inline-flex items-center disabled:cursor-not-allowed aria-disabled:cursor-not-allowed disabled:opacity-75 aria-disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-inverted bg-primary hover:bg-primary/75 disabled:bg-primary aria-disabled:bg-primary focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-primary">
                <!--v-if--> Check Backlink
                <!--v-if-->
              </button><span data-v-8dd619f3="">(must have a <bold data-v-8dd619f3="">do follow</bold> backlink to current site on the submission domain homepage, if not working, ensure you have \`SERPVerificationBot/1.0\` user-agent whitelisted)</span></div>
          </div>
          <form data-v-8dd619f3="" id="v-0-0-0" class="space-y-6">
            <div data-v-8dd619f3="" class="text-sm mt-6">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-1" class="block font-medium text-default after:content-['*'] after:ms-0.5 after:text-error">Company Name</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-1" type="text" name="name" placeholder="SERP AI" class="w-full rounded-md border-0 placeholder:text-dimmed focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-highlighted bg-default ring ring-inset ring-accented focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-primary" autocomplete="off" aria-describedby="" aria-invalid="false">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-2" class="block font-medium text-default after:content-['*'] after:ms-0.5 after:text-error">Domain</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-2" type="text" name="domain" placeholder="serp.ai" class="w-full rounded-md border-0 placeholder:text-dimmed focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-highlighted bg-default ring ring-inset ring-accented focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-primary" autocomplete="off" aria-describedby="" aria-invalid="false">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-3" class="block font-medium text-default after:content-['*'] after:ms-0.5 after:text-error">Description</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><textarea id="v-0-0-3" name="description" rows="5" placeholder="A detailed overview of your company" class="w-full rounded-md border-0 placeholder:text-dimmed focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-highlighted bg-default ring ring-inset ring-accented focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-primary" aria-describedby="" aria-invalid="false"></textarea>
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-4" class="block font-medium text-default after:content-['*'] after:ms-0.5 after:text-error">Pricing Options</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <!---->
                <div dir="ltr" id="v-0-0-4" class="relative inline-flex items-center flex-wrap w-full rounded-md transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-highlighted bg-default ring ring-inset ring-accented has-focus-visible:ring-2 has-focus-visible:ring-primary focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-primary pe-9"><input type="text" autocomplete="false" autocorrect="off" autocapitalize="off" aria-describedby="" aria-invalid="false" class="border-0 bg-transparent placeholder:text-dimmed focus:outline-none disabled:cursor-not-allowed disabled:opacity-75" aria-disabled="false" aria-expanded="false" aria-controls="" aria-autocomplete="list" role="combobox" dir="ltr" value="">
                  <!---->
                  <!--v-if--><button type="button" tabindex="-1" aria-label="Show popup" aria-haspopup="listbox" aria-expanded="false" aria-controls="" data-state="closed" aria-disabled="false" class="group absolute inset-y-0 end-0 flex items-center disabled:cursor-not-allowed disabled:opacity-75 pe-2.5"><span class="iconify i-lucide:chevron-down shrink-0 text-dimmed size-5" aria-hidden="true"></span></button>
                </div>
                <!--teleport start-->
                <!--teleport end-->
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-5" class="block font-medium text-default after:content-['*'] after:ms-0.5 after:text-error">Tagline</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-5" type="text" name="oneLiner" placeholder="A short company tagline (max 75 characters)" class="w-full rounded-md border-0 placeholder:text-dimmed focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-highlighted bg-default ring ring-inset ring-accented focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-primary" autocomplete="off" maxlength="75" aria-describedby="" aria-invalid="false">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-6" class="block font-medium text-default">Category(s)</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <!---->
                <div dir="ltr" id="v-0-0-6" class="relative inline-flex items-center flex-wrap w-full rounded-md transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-highlighted bg-default ring ring-inset ring-accented has-focus-visible:ring-2 has-focus-visible:ring-primary focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-primary pe-9"><input type="text" autocomplete="false" autocorrect="off" autocapitalize="off" aria-describedby="" aria-invalid="false" class="border-0 bg-transparent placeholder:text-dimmed focus:outline-none disabled:cursor-not-allowed disabled:opacity-75" aria-disabled="false" aria-expanded="false" aria-controls="" aria-autocomplete="list" role="combobox" dir="ltr" value="">
                  <!---->
                  <!--v-if--><button type="button" tabindex="-1" aria-label="Show popup" aria-haspopup="listbox" aria-expanded="false" aria-controls="" data-state="closed" aria-disabled="false" class="group absolute inset-y-0 end-0 flex items-center disabled:cursor-not-allowed disabled:opacity-75 pe-2.5"><span class="iconify i-lucide:chevron-down shrink-0 text-dimmed size-5" aria-hidden="true"></span></button>
                </div>
                <!--teleport start-->
                <!--teleport end-->
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-7" class="block font-medium text-default">Tags</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-7" type="text" name="tags" placeholder="e.g. AI, SEO, Marketing" class="w-full rounded-md border-0 placeholder:text-dimmed focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-highlighted bg-default ring ring-inset ring-accented focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-primary" autocomplete="off" aria-describedby="" aria-invalid="false" value="">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-8" class="block font-medium text-default">Company Logo</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="rounded border border-dashed border-gray-300 p-4"><input data-v-8dd619f3="" type="file" accept="image/*" class="w-full text-sm">
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-center"><button type="submit" class="rounded-md font-medium inline-flex items-center disabled:cursor-not-allowed aria-disabled:cursor-not-allowed disabled:opacity-75 aria-disabled:opacity-75 transition-colors px-3 py-2 text-base gap-2 text-inverted bg-secondary hover:bg-secondary/75 disabled:bg-secondary aria-disabled:bg-secondary focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-secondary">
                <!--v-if--> Submit
                <!--v-if-->
              </button></div>
          </form>
        </div>
        <!--v-if-->
      </div>
    </main>
  </div>
  <!--v-if-->
</div>"
`;

exports[`SPage/Users/Submit/Company Snapshot > renders New Submission (no id) correctly 1`] = `
"<div data-v-8dd619f3="" class="flex-col lg:grid lg:grid-cols-10 lg:gap-10 flex min-h-screen items-center justify-center">
  <!--v-if-->
  <div class="lg:col-span-10">
    <main data-v-8dd619f3="" class="min-h-[calc(100vh-var(--ui-header-height))] py-20">
      <div data-v-8dd619f3="" class="rounded-lg bg-default ring ring-default divide-y divide-default mx-auto max-w-2xl p-8">
        <!--v-if-->
        <div class="p-4 sm:p-6">
          <uheading data-v-8dd619f3="" level="2" class="mb-6 text-center text-2xl font-semibold"> Submit Company </uheading>
          <!--v-if-->
          <form data-v-8dd619f3="" id="v-0-0-0" class="space-y-6">
            <div data-v-8dd619f3="" class="text-sm mt-6">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-1" class="block font-medium text-default after:content-['*'] after:ms-0.5 after:text-error">Company Name</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-1" type="text" name="name" placeholder="SERP AI" class="w-full rounded-md border-0 placeholder:text-dimmed focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-highlighted bg-default ring ring-inset ring-accented focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-primary" autocomplete="off" aria-describedby="" aria-invalid="false" value="">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-2" class="block font-medium text-default after:content-['*'] after:ms-0.5 after:text-error">Domain</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-2" type="text" name="domain" placeholder="serp.ai" class="w-full rounded-md border-0 placeholder:text-dimmed focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-highlighted bg-default ring ring-inset ring-accented focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-primary" autocomplete="off" aria-describedby="" aria-invalid="false" value="">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-3" class="block font-medium text-default after:content-['*'] after:ms-0.5 after:text-error">Description</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><textarea id="v-0-0-3" name="description" rows="5" placeholder="A detailed overview of your company" class="w-full rounded-md border-0 placeholder:text-dimmed focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-highlighted bg-default ring ring-inset ring-accented focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-primary" aria-describedby="" aria-invalid="false" value=""></textarea>
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-4" class="block font-medium text-default after:content-['*'] after:ms-0.5 after:text-error">Pricing Options</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <!---->
                <div dir="ltr" id="v-0-0-4" class="relative inline-flex items-center flex-wrap w-full rounded-md transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-highlighted bg-default ring ring-inset ring-accented has-focus-visible:ring-2 has-focus-visible:ring-primary focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-primary pe-9"><input type="text" autocomplete="false" autocorrect="off" autocapitalize="off" aria-describedby="" aria-invalid="false" class="border-0 bg-transparent placeholder:text-dimmed focus:outline-none disabled:cursor-not-allowed disabled:opacity-75" aria-disabled="false" aria-expanded="false" aria-controls="" aria-autocomplete="list" role="combobox" dir="ltr" value="">
                  <!---->
                  <!--v-if--><button type="button" tabindex="-1" aria-label="Show popup" aria-haspopup="listbox" aria-expanded="false" aria-controls="" data-state="closed" aria-disabled="false" class="group absolute inset-y-0 end-0 flex items-center disabled:cursor-not-allowed disabled:opacity-75 pe-2.5"><span class="iconify i-lucide:chevron-down shrink-0 text-dimmed size-5" aria-hidden="true"></span></button>
                </div>
                <!--teleport start-->
                <!--teleport end--><input data-hidden="" tabindex="-1" style="position: absolute; border: 0px; width: 1px; height: 1px; padding: 0px; margin: -1px; overflow: hidden; clip: rect(0, 0, 0, 0); clip-path: inset(50%); white-space: nowrap; word-wrap: normal;" name="pricing" value="">
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-5" class="block font-medium text-default after:content-['*'] after:ms-0.5 after:text-error">Tagline</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-5" type="text" name="oneLiner" placeholder="A short company tagline (max 75 characters)" class="w-full rounded-md border-0 placeholder:text-dimmed focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-highlighted bg-default ring ring-inset ring-accented focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-primary" autocomplete="off" maxlength="75" aria-describedby="" aria-invalid="false" value="">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-6" class="block font-medium text-default">Category(s)</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <!---->
                <div dir="ltr" id="v-0-0-6" class="relative inline-flex items-center flex-wrap w-full rounded-md transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-highlighted bg-default ring ring-inset ring-accented has-focus-visible:ring-2 has-focus-visible:ring-primary focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-primary pe-9"><input type="text" autocomplete="false" autocorrect="off" autocapitalize="off" aria-describedby="" aria-invalid="false" class="border-0 bg-transparent placeholder:text-dimmed focus:outline-none disabled:cursor-not-allowed disabled:opacity-75" aria-disabled="false" aria-expanded="false" aria-controls="" aria-autocomplete="list" role="combobox" dir="ltr" value="">
                  <!---->
                  <!--v-if--><button type="button" tabindex="-1" aria-label="Show popup" aria-haspopup="listbox" aria-expanded="false" aria-controls="" data-state="closed" aria-disabled="false" class="group absolute inset-y-0 end-0 flex items-center disabled:cursor-not-allowed disabled:opacity-75 pe-2.5"><span class="iconify i-lucide:chevron-down shrink-0 text-dimmed size-5" aria-hidden="true"></span></button>
                </div>
                <!--teleport start-->
                <!--teleport end-->
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-7" class="block font-medium text-default">Tags</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-7" type="text" name="tags" placeholder="e.g. AI, SEO, Marketing" class="w-full rounded-md border-0 placeholder:text-dimmed focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-highlighted bg-default ring ring-inset ring-accented focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-primary" autocomplete="off" aria-describedby="" aria-invalid="false" value="">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-8" class="block font-medium text-default">Company Logo</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="rounded border border-dashed border-gray-300 p-4"><input data-v-8dd619f3="" type="file" accept="image/*" class="w-full text-sm">
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-center"><button type="submit" class="rounded-md font-medium inline-flex items-center disabled:cursor-not-allowed aria-disabled:cursor-not-allowed disabled:opacity-75 aria-disabled:opacity-75 transition-colors px-3 py-2 text-base gap-2 text-inverted bg-secondary hover:bg-secondary/75 disabled:bg-secondary aria-disabled:bg-secondary focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-secondary">
                <!--v-if--> Submit
                <!--v-if-->
              </button></div>
          </form>
        </div>
        <!--v-if-->
      </div>
    </main>
  </div>
  <!--v-if-->
</div>"
`;
